Apache-Arrow-Flight-Tester/main.py

```diff
@@ -90,6 +90,42 @@ def list_actions(flight_client):
     print(list(response))
 
 
+def update_object_store_to_s3(flight_client, endpoint, bucket_name, access_key_id, secret_access_key):
+    object_store_type_bytes = str.encode("s3")
+    object_store_type_size = len(object_store_type_bytes).to_bytes(2, byteorder="big")
+
+    arguments = [endpoint, bucket_name, access_key_id, secret_access_key]
+    action_body = object_store_type_size + object_store_type_bytes + create_update_object_store_action_body(arguments)
+
+    result = flight_client.do_action(pyarrow.flight.Action("UpdateRemoteObjectStore", action_body))
+
+    print(list(result))
+
+
+def update_object_store_to_azure_blob_storage(flight_client, account, access_key, container_name):
+    object_store_type_bytes = str.encode("azureblobstorage")
+    object_store_type_size = len(object_store_type_bytes).to_bytes(2, byteorder="big")
+
+    arguments = [account, access_key, container_name]
+    action_body = object_store_type_size + object_store_type_bytes + create_update_object_store_action_body(arguments)
+
+    result = flight_client.do_action(pyarrow.flight.Action("UpdateRemoteObjectStore", action_body))
+
+    print(list(result))
+
+
+def create_update_object_store_action_body(arguments):
```

Comment:
Maybe prepend the object store type to arguments. As far as I can tell the object store type and argument are encoded in the same manner. If this change is made it is probably better keep to the code simple and just use two byte for everything instead of adding additional complexity to allow each argument to be encoded with a different size.

Reply:
Fixed.